### ISSUE TITLE
Is this lib still maintained ?

### ISSUE THREAD

**Issue by stychu** [rhalff/dot-object#86]
I see few issues not being tackled, latest release failing. Is this project still maintained ?
Any good alternative if so ?

**Comment by rhalff** [comment]
@stychu what is failing?

**Comment by stychu** [comment]
Just checking coz I've seen reports in issues regarding some old dependencies which cause possible memory leaks

**Comment by MattCollins84** [comment]
Yes, the `inflight` package which is a dependency of `glob@7.1.6` causes memory leaks - it would be good to get rid of that!

**Comment by MikeMcC399** [comment]
This is also being reported by

https://security.snyk.io/vuln/SNYK-JS-INFLIGHT-6095116
Missing Release of Resource after Effective Lifetime

- See also dedicated issue https://github.com/rhalff/dot-object/issues/83

**Comment by MikeMcC399** [comment]
@stychu 

You raise a good question! Would you like to edit the title, which says "Is this lib maintained still maintained ?" which has got the word "maintained" in it once too much.

**Comment by AQS-DTheuke** [comment]
Installing this package displays warnings about deprecated dependencies.
Could you please create a new release with up to date dependencies?

**Comment by MikeMcC399** [comment]
[@actions/cache](https://github.com/actions/toolkit/tree/main/packages/cache) assumed that this package is unmaintained and removed the dependency on it by migrating away from [twirp-ts](https://github.com/hopin-team/twirp-ts) which in turn depends on `dot-object` here

**Comment by AQS-DTheuke** [comment]
I now started to receive warnings about a CVE introduced via dependencies of this lib.

````txt
brace-expansion@1.1.16
└─┬ minimatch@3.1.5
  └─┬ glob@7.2.3
    └─┬ dot-object@2.1.5
      └─┬ ...
````

`brace-expansion@1.1.16` <- [CVE-2026-14257](https://nvd.nist.gov/vuln/detail/CVE-2026-14257).

Would it be possible to release a patched version?
